spec/rspec/expectations/fail_with_spec.rb

```diff
@@ -47,7 +47,6 @@
       |@@ -1,2 +1,2 @@
       |-["poo", "car"]
       |+[(a string matching /foo/), (a string matching /bar/)]
-      |
```

Comment:
We're not (generally) interested in reducing allocations in our own test suite, and I'm pretty certain this is a change in behaviour, can you restore it?